README.md

```diff
@@ -15,16 +15,14 @@
 
 
 
-## Build Status
-
-[![master Build Status](https://travis-ci.org/AshamaneProject/AshamaneCore.svg?branch=master)](https://travis-ci.org/AshamaneProject/AshamaneCore)
-[![Build status](https://ci.appveyor.com/api/projects/status/kb0wjxh3va9c3e0k/branch/master?svg=true)](https://ci.appveyor.com/project/Traesh/ashamanecore/branch/master)
+## Build Status 
+[![master Build Status](https://travis-ci.org/Daniel25/AshamaneCore.svg?branch=master)](https://travis-ci.org/github/Daniel25/AshamaneCore)
```

Comment:
When you do a pull request to the Master Repo, try to don´t request your readme changes.

Reply:
That happens when someone create a pull requestthe other repository which isnt his xD

Reply:
Oh xD wasn´t aware of this, just quickly checked the code, for usefull stuff XD

Reply:
> Oh xD wasn´t aware of this, just quickly checked the code, for usefull stuff XD

It's ok,not much usefull stuff there xD